continuous-deployment/readme.md

```diff
@@ -1,7 +1,142 @@
-# Continuous Deployment
+# Continuous Delivery
 
-Please see [CI/CD](../continuous-integration/CICD.md) while this page is awaiting contributions.
+The inspiration behind continuous delivery is constantly delivering valuable software to users and developers more frequently. Applying the principles and practices laid out in this readme will help you reduce risk, eliminate manual operations and increase quality and confidence.
 
-## Sections within Continuous Deployment
+Deploying software involves the following principles:
 
-* [Secrets Management](secrets-management/readme.md)
+1. Provision and manage the cloud environment runtime for your application (cloud resources, infrastructure, hardware, services, etc).
+
+1. Install the target application version across your cloud environments.
+
+1. Configure your application, including any required data.
+
+A continuous deployment pipeline is an automated manifestation of your process to streamline these very principles in a consistent and repeatable manner.
+
+## Goal
+
+* Follow industry best practices for delivering software changes to customers and developers.
+
+* Establish consistency for the guiding principles and best practices when assembling continuous delivery workflows.
+
+## General Guidance
+
+### Define a Release Strategy
+
+It's important to establish a common understanding between the technical lead and application stakeholder(s) around the release strategy / design  during the planning phase of a project. This common understanding includes the deployment and maintenance of the application throughout its SDLC.
+
+#### Release Strategy Principles
+
+*Continuous Delivery* by Jez Humble, David Farley cover the key considerations to follow when creating a release strategy:
```

Comment:
I would suggest adding a link to the book in the reference section. Would be helpful for the audience.

Reply:
Good catch. I'll add the book reference.